src/services/MessageRepository.tsx

```diff
@@ -246,7 +247,11 @@ class MessageRepository {
       case MessageBodyType.REQUEST_TERMS:
         return [(message.body as IRequestTerms).content.cTypeHash]
       case MessageBodyType.SUBMIT_TERMS:
-        return [(message.body as ISubmitTerms).content.claim.cTypeHash]
+        // Need to fix with the Message compress and decompress update
```

Comment:
What needs to be fixed? the `as IPartialClaim`?

Reply:
Yes, I have done this. I forgot to push.